api/client/agents.go

```diff
@@ -21,10 +21,12 @@ import (
 	"github.com/Mierdin/todd/hostresources"
 )
 
-// Agent will query the ToDD server for a list of currently registered agents, and will display
+// Agents will query the ToDD server for a list of currently registered agents, and will display
 // a list of them to the user. Optionally, the user can provide a subargument containing the UUID of
 // a registered agent, and this function will output more detailed information about that agent.
-func (capi ClientApi) Agents(conf map[string]string, agentUuid string) {
+func (capi ClientApi) Agents(conf map[string]string, agentUuid string) (error, []defs.AgentAdvert) {
```

Comment:
Convention is for the error to be the last return value (ie, `([]defs.AgentAdvert, error)`